tests/data/fields/label_field_test.py

```diff
@@ -1,17 +1,19 @@
 # pylint: disable=no-self-use,invalid-name
 import numpy
+import pytest
 
 from allennlp.data.vocabulary import Vocabulary
 from allennlp.data.fields import LabelField
 from allennlp.common.testing import AllenNlpTestCase
+from allennlp.common.checks import ConfigurationError
 
 
 class TestLabelField(AllenNlpTestCase):
 
-    def test_pad_returns_one_hot_array(self):
-        label = LabelField(5, num_labels=10)
+    def test_pad_returns_integer_array(self):
```

Comment:
Looks like we missed renaming the tests themselves when we switched from `.pad()` to `.as_array()`.  Can you fix this one, at least?